scripts/ia_healthcheck.py

```diff
@@ -0,0 +1,92 @@
+from datetime import datetime, timedelta
+import random
+from ..web_monitoring import db
+from ..web_monitoring import internetarchive
+
+
+MAX_CAPTURE_AGE = timedelta(hours=72)
+LINKS_TO_CHECK = 5
+
+
+def query_webmondb():
+    """
+    Query the web monitoring SCANNER API to get random links from the database
+    Interger -> List
+    """
+    try:
+        response = db.Client.from_env()
+    except Exception as e:
+        print('Could not access Web Monitoring API')
+        print(e)
+        return []
```

Comment:
Is there really any point in returning and going any further in this case? Maybe better to just let the exception crash the program.

Otherwise, if you want to suppress all the normal logging when the client is not configured, I’d suggest handling *specifically* the `MissingCredentials` error and letting all others crash, since they’re unexpected and you’ll want the full readout:

```py
except db.MissingCredentials as error:
    print(error)
    return []
```

Note I don’t think you really need a custom message; the `MissingCredentials` one is already pretty fancy: https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/master/web_monitoring/db.py#L154-L162

Reply:
Removed custom message and added MissingCredentials.